libs/pipelines/dod_pipeline.py

```diff
@@ -0,0 +1,87 @@
+from typing import Tuple
+from collections import namedtuple
+import logging
+
+from libs.enums import Intervention
+from libs import validate_results
+from libs import build_dod_dataset
+from libs import dataset_deployer
+from libs.functions import generate_shapefiles
+
+logger = logging.getLogger(__name__)
+PROD_BUCKET = "data.covidactnow.org"
+
+DodInterventionResult = namedtuple(
+    "DodInterventionResult", ["key", "projection_df", "shapefile_data"]
+)
+
+
+def run_projections(
+    input_file, intervention: Intervention, run_validation=True
+) -> Tuple[DodInterventionResult, DodInterventionResult]:
```

Comment:
it would be good to know which one is for state or county, or if can be abstracted by geos

Reply:
yeah i originally had this as "run_county_projections" and "run_state_projections", would you prefer that? I dont' think we're quite stable enough to have the same code runs on just a different aggregation